README.md

```diff
@@ -25,14 +25,14 @@ The app launches a window displaying a map.
 2. In the import wizard, choose _Maven > Existing Maven Projects_, then click _Next_.
 3. Select the java-maven-starter-project as the project root directory.
 4. Click _Finish_ to complete the import.
-5. Right-click the project in the Project Explorer or Package Explorer and choose _Run As > Maven Build..._. In the _Edit Configuration_ dialog, create a new configuration with name `arcgis`. In the _Goals_ field, enter `arcgis-java:arcgis`. Click _Run_ to run the goal. This will cause the ArcGIS Runtime SDK Maven plugin to download the native libraries.
+5. Right-click the project in the Project Explorer or Package Explorer and choose _Run As > Maven Build..._. In the _Edit Configuration_ dialog, create a new configuration with name `unpack`. In the _Goals_ field, enter `dependency:unpack`. Click _Run_ to run the goal. This will unpack the native libraries into $USER_HOME/.arcgis.
 6. Again, create a new run configuration with name `run`. In the _Goals_ field, enter `compile exec:java`. Click _Run_ to run the goal. The app should compile and launch the JavaFX window.
 
 ### Command Line
```

Comment:
Verified sample builds and runs on Git **Bash** for Windows command line when I changed `arcgis.version` to 100.5.0 in `pom.xml`. Need to re-verify at Update6 release.

Verfied sample builds and runs on Windows 10 Command Prompt when I changed `arcgis.version` to 100.5.0 in `pom.xml`. Need to re-verify at Update6 release.